src/template/header.html.twig

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
You should rename your file as "layout.html.twig", since you call both the <head> & <body>.